src/transactions/ManageOfferOpFrame.cpp

```diff
@@ -164,6 +167,8 @@ ManageOfferOpFrame::doApply(Application& app, LedgerDelta& delta,
             return false;
         }
 
+        mSellSheepOffer->storeDelete(tempDelta, db);
```

Comment:
I would prefer keeping calls to `storeDelete` with their `addNumEntries` counter part so that the account state is valid during execution